cypress/features/regression/accessibility/accessibilityCommon.feature

```diff
@@ -36,7 +36,7 @@ Feature: Accessibility tests - Common list
     When I open "Button-Toggle-Group" component page "basic" in no iframe
     Then "Button Toggle Group" component has no accessibility violations
 
-  @ignore
+  @accessibility
```

Comment:
Could be worth removing this all together as it is no longer testing anything? I think anyway

Reply:
It's looping over a load of components with the `component` example I think - I will remove the comments about being ignored though, missed those

Reply:
Actually no you're right, maybe it isn't doing anything now